### ISSUE TITLE
progressive webapp at a wedding - it's fun

### ISSUE THREAD

**Issue by lyckligtax** [nodeconf/oslo#38]
### TL;DR
- progressive web apps
- multimedia
- web components with Polymer
- Node.js backends
- cheating the system by not paying a photographer for a wedding
### Background

My beloved fiancée and I are planning our perfect wedding for April this year. We of course, also discussed who is going to record this unique event in sound and vision. However, as many young couples, we are a little short on money and cannot really afford an expensive, professional photographer who charges 900 € for 10 pictures.

Does that mean we have to do without memorable pictures of our great day?

No. The solution is simple. Nowadays, in our society, almost everybody owns a smartphone and most smartphones are equipped with excellent cameras. Also, people do use them, constantly, documenting every titbit of their lives.

Thus, my fiancée and I plan to make use of that social phenomenon and add to it my skills as a professional developer.

We decided to provide a clean interface that will encourage our friends to take a hell of a lot of photos of the party. We will let our guests take pictures, videos or audio messages and will share them instantly across all devices. What a great benefit!
### The talk

My talk is about progressive web apps and how to make the most of them. The audience is shown how to leverage web components in order to create a neat interface across different platforms. All backed by Node.js and the use of cool multimedia APIs.

The audience will judge the results based on some cool pictures I will hopefully present and also, will be able to test the service hands-on.
### Bio

My name is Tobias Subklewe and I work as lead Managed Node.js Developer at maxcluster GmbH, one of the largest European providers of hosting solutions for e-commerce companies. My team and I develop crucial infrastructure and management tools for over 500 customers and more than 600 clusters, thereby using state of the art Node.js.
